common/service/dynamicconfig/clientInterface.go

```diff
@@ -45,3 +47,7 @@ type Client interface {
 	// UpdateValue takes value as map and updates by overriding. It doesn't support update with filters.
 	UpdateValue(name Key, value interface{}) error
 }
+
+var notFoundError = &shared.EntityNotExistsError{
```

Comment:
As we are migrating to internal types. Can we use types defined in `common/types` instead?

Reply:
Yeah good point. almost forgot that.